src/Main.cpp

```diff
@@ -96,6 +98,20 @@ int main(int argc, char** argv)
         }
         std::cout << wp_device.getConfig() << std::endl;
     }
+    else if (cmd == "set-id") {
+        if (argc != 7)
+        {
+            std::cerr << "wrong number of arguments" << std::endl;
+        }
+        power_whisperpower::Device device(device_group, device_id);
+        waitResult(*can_device, device, device.querySerialNumber());
+        uint32_t serial_number = device.getSerialNumber();
+        int device_new_id(std::stoi(argv[6]));
+        waitResult(*can_device, device, device.querySetId(0x600, serial_number));
```

Comment:
```suggestion
        power_whisperpower::Device wp_device(device_group, 0x600);
        waitResult(*can_device, wp_device, wp_device.querySerialNumber());
        uint32_t serial_number = device.getSerialNumber();
        int device_new_id(std::stoi(argv[5]));
        waitResult(*can_device, wp_device, device.querySetId(device_new_id, serial_number));
```

also write a disclaimer at the helpers stating that when using set-id; the `DEVICE_ID` argument is actually the _NEW DEVICE ID_ to be set